src/uct/ib/rdmacm/rdmacm_cm_ep.c

```diff
@@ -158,15 +158,15 @@ UCS_CLASS_INIT_FUNC(uct_rdmacm_cm_ep_t, const uct_ep_params_t *params)
     if (!(params->field_mask & (UCT_EP_PARAM_FIELD_SOCKADDR |
                                 UCT_EP_PARAM_FIELD_CONN_REQUEST))) {
         ucs_error("neither UCT_EP_PARAM_FIELD_SOCKADDR nor "
-                  "UCT_EP_PARAM_FIELD_CONN_REQUEST is set. ep params: %zu",
+                  "UCT_EP_PARAM_FIELD_CONN_REQUEST is set. field_mask 0x%lx",
                   params->field_mask);
         return UCS_ERR_INVALID_PARAM;
     }
 
-    self->cm = ucs_derived_of(params->cm, uct_rdmacm_cm_t);
-
-    UCS_CLASS_CALL_SUPER_INIT(uct_base_ep_t, &self->cm->super.iface);
+    cm = ucs_derived_of(params->cm, uct_rdmacm_cm_t);
+    UCS_CLASS_CALL_SUPER_INIT(uct_base_ep_t, &cm->super.iface);
```

Comment:
1. pass params->cm->iface to SUPER_INIT
2. in line 169, set self->cm = ucs_derived_of(params->cm, uct_rdmacm_cm_t);
3. next PR: remove the field self->cm. resolve endpoint's cm by ep->super.iface